guides/cookbook.md

```diff
@@ -103,6 +103,29 @@ router.get(
 )
 ```
 
+#### Altering all responses
+
+You can also write catch-all route that will then alter all the responses. One example where this is useful would be injecting [Content Security Policy](/guides/security#section_content_security_policy__csp_).
```

Comment:
```suggestion
You can also write catch-all route that will then alter all the responses. One example where this is useful would be injecting [Content Security Policy](security#section_content_security_policy__csp_).
```

Reply:
Make these relative links to preserve the version in the URL when present.

Reply:
Got it.